db/datagenerator.js

```diff
@@ -0,0 +1,106 @@
+/*
+1. Take existing photos from the database and perform heap permutation
+2. Throlle csv file creation of the 80 to 83 photos found in the database
+*/
+const permutations = require("./heapperm.js");
+const createCsvWriter = require('csv-writer').createArrayCsvWriter;
+const csvWriter = createCsvWriter({
+    path: `${__dirname}/inputfile.csv`,
+    append: true
+});
+const mysql = require("mysql");
+const connection = mysql.createConnection({
+  host: "localhost",
+  user: "root",
+  password: "MyNewPass",
+  database: "airbnb"
+});
+var dataset = [], writeMode = true; // writeMode true means Write New, false means Append
+var sequence = permutations.rs;
+//console.log(sequence);
+connection.connect();
```

Comment:
For cleanliness's sake I'd consider splitting this into two separate files - one for the connection, which you can then export, and one for the data generator functions.